Source/SwiftLintBuiltInRules/Rules/Style/TrailingClosureRule.swift

```diff
@@ -92,10 +166,91 @@ private extension FunctionCallExprSyntax {
         }
         return nil
     }
+
+    func dropLastArgument() -> Self {
+        let arguments = LabeledExprListSyntax(arguments.dropLast())
+
+        return self
+            .with(\.arguments, arguments.dropLastTrailingComma())
+            .dropParensIfEmpty()
+    }
+
+    func dropParensIfEmpty() -> Self {
+        if arguments.isEmpty {
+            self
+                .with(\.rightParen, nil)
+                .with(\.leftParen, nil)
+        } else {
+            self
+        }
+    }
+
+    func convertToTrailingClosure() -> Self? {
+        guard trailingClosure == nil, let lastDistinctClosureParameter else { return nil }
+        let leadingTrivia = lastTriviasInArguments?
+            .removingLeadingNewlinesIfExists()
+            .appendingSpaceIfNoTrailingSpace() ?? []
+
+        return dropLastArgument()
+            .with(\.trailingClosure, lastDistinctClosureParameter.with(\.leadingTrivia, leadingTrivia))
+            .with(\.calledExpression.trailingTrivia, [])
+    }
+
+    var lastTriviasInArguments: Trivia? {
```

Comment:
Trivia is a plural noun:
```suggestion
    var lastTriviaInArguments: Trivia? {
```